hudi-common/src/main/java/org/apache/hudi/common/util/FileIOUtils.java

```diff
@@ -160,4 +160,33 @@ public static void closeQuietly(Closeable closeable) {
       LOG.warn("IOException during close", e);
     }
   }
+
+  public static void createFileInPath(FileSystem fileSystem, org.apache.hadoop.fs.Path fullPath, Option<byte[]> content) {
```

Comment:
nit: have `throws HoodieIOException` in the method signature?

Reply:
Sure. Changed.